mod_ci/controllers.py

```diff
@@ -442,6 +442,24 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
             platform=platform, id=test.id))
 
 
+def ci_badge(state):
+    if test_type != TestType.pull_request:
+        if state == Status.SUCCESS:
+            with open('../static/badges/Test-Passed-brightgreen.svg',
+                      'r') as svg:
+                return svg.read()
```

Comment:
What's the point of doing a return if you don't do anything with that return?

Reply:
i guess i have to change read into something else